server/tests/db.test.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+
+beforeAll(async () => {
+  await mongoose.connect('mongodb://localhost/gameinfo', { useNewUrlParser: true, useUnifiedTopology: true });
+});
+
+describe('Mongoose', () => {
```

Comment:
Nice job testing out your DB connection, feel it may be unnecessary to test that but it is a nice to have while running your tests